src/utils/isRadioOrCheckbox.ts

```diff
@@ -0,0 +1,6 @@
+import isRadioInput from './isRadioInput';
+import isCheckBoxInput from './isCheckBoxInput';
+import { FieldElement } from '../types';
+
+export default (ref: FieldElement): boolean =>
```

Comment:
From the point of view of type guard, we think that we should:

```suggestion
export default (ref: FieldElement): ref is HTMLInputElement =>
```